client/state/selectors/get-inline-help-support-variation.js

```diff
@@ -0,0 +1,39 @@
+/** @format */
+/**
+ * Internal dependencies
+ */
+import config from 'config';
+import { getCurrentUserLocale } from 'state/current-user/selectors';
+import { isDirectlyFailed } from 'state/selectors';
+import isHappychatAvailable from 'state/happychat/selectors/is-happychat-available';
+import isHappychatUserEligible from 'state/happychat/selectors/is-happychat-user-eligible';
+import { isTicketSupportEligible } from 'state/help/ticket/selectors';
+
+const SUPPORT_DIRECTLY = 'SUPPORT_DIRECTLY';
+const SUPPORT_HAPPYCHAT = 'SUPPORT_HAPPYCHAT';
+const SUPPORT_TICKET = 'SUPPORT_TICKET';
+const SUPPORT_FORUM = 'SUPPORT_FORUM';
+
+/**
+ * @param {Object} state Global state tree
+ * @return {String} True if current user is able to see the checklist after checkout
```

Comment:
I'll need to update this doc entry

Reply:
Yep! 😄

Reply:
Oops, I thought I'd already pushed this change - done now :)